tests/bgp/test_bgpmon.py

```diff
@@ -5,33 +5,70 @@
 from jinja2 import Template
 import ptf.packet as scapy
 from ptf.mask import Mask
+import json
 from tests.common.fixtures.ptfhost_utils import change_mac_addresses      # lgtm[py/unused-import]
 from tests.common.fixtures.ptfhost_utils import remove_ip_addresses       # lgtm[py/unused-import]
 from tests.common.helpers.generators import generate_ips as generate_ips
+from tests.common.helpers.assertions import pytest_assert
 
 pytestmark = [
     pytest.mark.topology('any'),
 ]
 
-BGPMON_TEMPLATE_FILE = 'bgp/bgpmon.j2'
+BGPMON_TEMPLATE_FILE = 'bgp/templates/bgpmon.j2'
 BGPMON_CONFIG_FILE = '/tmp/bgpmon.json'
 BGP_PORT = 179
-BGP_CONNECT_TIMEOUT = 120
+BGP_CONNECT_TIMEOUT = 121
+ZERO_ADDR = r'0.0.0.0/0'
 
 logger = logging.getLogger(__name__)
 
+def route_through_default_routes(host, ip_addr):
+    output = host.shell("show ip route {} json".format(ip_addr))['stdout']
+    routes_info = json.loads(output)
+    ret = True
+    for prefix in routes_info.keys():
+        if prefix != ZERO_ADDR:
+            ret = False
+            break
+    return ret
+
+def generate_ip_through_default_route(host):
+    # Generate an IP address routed through default routes
+    ip_addrs = generate_ips(200, "200.0.0.1/24", [])
```

Comment:
This code will iterate through the same 200.0.0.0/24 subnet.
So if the first prefix fail, other prefixes would fail too.

Reply:
Updated. Thanks!